tests/test_environment.py

```diff
@@ -8,29 +8,30 @@
 from flow.environment import TestEnvironment
 from flow.errors import ConfigKeyError
 from test_project import StringIO, redirect_stdout
+import pytest
 
 
 class ProjectTest(unittest.TestCase):
```

Comment:
``` suggestion
 class TestProject():

Reply:
Instead of inheriting from `unittest.TestCase`  you can change the name of the class  starting from `Test`.

Reply:
Thanks for the advice, will do it.